src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -112,5 +112,5 @@ jobs:
       {% raw -%}
       - name: Update .pot files
         run: oca_export_and_push_pot https://x-access-token:${{ secrets.GIT_PUSH_TOKEN }}@github.com/${{ github.repository }}
-        if: ${{ matrix.makepot == 'true' && github.event_name == 'push' && github.repository_owner == 'OCA' }}
+        if: ${{ matrix.makepot == 'true' && github.event_name == 'push' && github.repository_owner == {{ org_slug }} }}
```

Comment:
Here we are in a `raw` section, so the org slug is not replaced.
The raw section is to avoid escaping the double moustaches of GitHub expression so they are not interpreted by jinja. 
I think I did not have the raw section before you can look in the history. It's gonna be a little bit messy but it's feasible.

Reply:
Thanks ! Done.